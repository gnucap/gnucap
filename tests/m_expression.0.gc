option list
eval "string"
param a="aaa"
eval a
param b={"aaa" + "bbb"}
eval b
param c={"aaa" == "bbb"}
eval c
param d={"aaa" == a}
eval d
param e={exp("0.")}
eval e
param f={exp(1.+"0.")}
eval f
param g={exp("aaa" == 0.)}
eval g
param h={exp(0. == "aaa")}
eval h
param i={exp(a == "aaa")}
eval i
param j={exp(0. != 0.)}
eval j
param k={"aaa bbb"}
eval k
* //BUG// does not work with spice get_line
param l={"aaa\"bbb"}
eval l
* //BUG// totally does not work with spice get_line
* param m=exp("aaa\"bbb")
* eval m
options nag
param n={"missing quote}
eval n
