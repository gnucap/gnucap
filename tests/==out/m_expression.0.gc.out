"eval "string" "
"string"="string"
"param a="aaa" "
"eval a "
a=aaa
"param b={"aaa" + "bbb"} "
"eval b "
b=("bbb" + "aaa")
"param c={"aaa" == "bbb"} "
"eval c "
c= 0.
"param d={"aaa" == a} "
"eval d "
d=("aaa" == aaa)
"param e={exp("0.")} "
"eval e "
e=exp("0.")
"param f={exp(1.+"0.")} "
"eval f "
f=exp(("0." +  1.))
"param g={exp("aaa" == 0.)} "
"eval g "
g=exp(("aaa" ==  0.))
"param h={exp(0. == "aaa")} "
"eval h "
h=exp(( 0. == "aaa"))
"param i={exp(a == "aaa")} "
"eval i "
i=exp((aaa == "aaa"))
"param j={exp(0. != 0.)} "
"eval j "
j= 1.
"param k={"aaa bbb"} "
"eval k "
eval k 
     ^ ? can't convert to float
"* //BUG// does not work with spice get_line "
"param l={"aaa\"bbb"} "
"eval l "
eval l 
     ^ ? can't convert to float
"* //BUG// totally does not work with spice get_line "
"* param m=exp("aaa\"bbb") "
"* eval m "
"options nag "
"param n={"missing quote} "
>>>>>param n={"missing quote} 
"eval n "
>>>>>eval n 
"missing quote
              ^ ? end of file in quoted string
"missing quote
^ ? string begins here
eval n 
     ^ ? can't convert to float
